Functions/Verbs/msvc90/script.js

```diff
@@ -0,0 +1,25 @@
+include(["Functions", "Engines", "Wine"]);
+include(["Functions", "Net", "Resource"]);
+include(["Functions", "Filesystem", "Extract"]);
+include(["Functions", "Verbs", "luna"]);
+
+Wine.prototype.msvc90 = function() {
```

Comment:
I'd like if possible to avoid hosting those files on playonlinux.com in the future. It is harder for external people like wine developers to know what exactly is behind. 
I guess we could try to extract the librairies from vcrun2008 redistributable and see if it works